Jellyfin.Server.Implementations/Users/UserManager.cs

```diff
@@ -0,0 +1,787 @@
+        {
+            if (user == null)
+            {
+                throw new ArgumentNullException(nameof(user));
+            }
+
+            await GetAuthenticationProvider(user).ChangePassword(user, newPassword).ConfigureAwait(false);
+            await UpdateUserAsync(user).ConfigureAwait(false);
+
+            OnUserPasswordChanged?.Invoke(this, new GenericEventArgs<User>(user));
+        }
+
+        /// <inheritdoc/>
+        public void ChangeEasyPassword(User user, string newPassword, string newPasswordSha1)
+        {
+            GetAuthenticationProvider(user).ChangeEasyPassword(user, newPassword, newPasswordSha1);
+            UpdateUser(user);
+
+            OnUserPasswordChanged?.Invoke(this, new GenericEventArgs<User>(user));
+        }
+
+        /// <inheritdoc/>
+        public UserDto GetUserDto(User user, string remoteEndPoint = null)
+        {
+            return new UserDto
+            {
+                Name = user.Username,
+                Id = user.Id,
+                ServerId = _appHost.SystemId,
+                HasPassword = user.Password != null,
+                EnableAutoLogin = user.EnableAutoLogin,
+                LastLoginDate = user.LastLoginDate,
+                LastActivityDate = user.LastActivityDate,
+                PrimaryImageTag = user.ProfileImage != null ? _imageProcessor.GetImageCacheTag(user) : null,
+                Configuration = new UserConfiguration
+                {
+                    SubtitleMode = user.SubtitleMode,
+                    HidePlayedInLatest = user.HidePlayedInLatest,
+                    EnableLocalPassword = user.EnableLocalPassword,
+                    PlayDefaultAudioTrack = user.PlayDefaultAudioTrack,
+                    DisplayCollectionsView = user.DisplayCollectionsView,
+                    DisplayMissingEpisodes = user.DisplayMissingEpisodes,
+                    AudioLanguagePreference = user.AudioLanguagePreference,
+                    RememberAudioSelections = user.RememberAudioSelections,
+                    EnableNextEpisodeAutoPlay = user.EnableNextEpisodeAutoPlay,
+                    RememberSubtitleSelections = user.RememberSubtitleSelections,
+                    SubtitleLanguagePreference = user.SubtitleLanguagePreference ?? string.Empty,
+                    OrderedViews = user.GetPreference(PreferenceKind.OrderedViews),
+                    GroupedFolders = user.GetPreference(PreferenceKind.GroupedFolders),
+                    MyMediaExcludes = user.GetPreference(PreferenceKind.MyMediaExcludes),
+                    LatestItemsExcludes = user.GetPreference(PreferenceKind.LatestItemExcludes)
+                },
+                Policy = new UserPolicy
+                {
+                    MaxParentalRating = user.MaxParentalAgeRating,
+                    EnableUserPreferenceAccess = user.EnableUserPreferenceAccess,
+                    RemoteClientBitrateLimit = user.RemoteClientBitrateLimit ?? -1,
```

Comment:
```suggestion
                    RemoteClientBitrateLimit = user.RemoteClientBitrateLimit ?? 0,
```

Legacy defaults to `0` if `RemoteClientBitrateLimit` is not set.

This is what is displayed upon user create using this PR currently
![firefox_KHXGICwSOM](https://user-images.githubusercontent.com/24963659/83977394-7f861380-a8bd-11ea-9c8d-b9334d727534.png)